src/server/checkBrowserVersion.js

```diff
@@ -0,0 +1,37 @@
+const childProcess = require('child_process');
+
+const firefox = require('selenium-webdriver/firefox');
+
+const config = require('./config');
+
+const MINIMUM_FIREFOX_VERSION = 50.0;
+const FIREFOX_VERSION_MATCHER = /Mozilla Firefox ([0-9.]+)/;
```

Comment:
As it is written, people will potentially see a misleading error message, which seems like it should be fixed.